estate/models/EstateProperty.py

```diff
@@ -0,0 +1,44 @@
+from datetime import datetime
+from odoo import models, fields
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
```

Comment:
```suggestion

    _name = 'estate.property'
```
Bad readability, needs a extra line in between the class and `_name` and make sure use `''` for technical strings and `""` for user readable strings